src/main/java/uk/gov/pay/adminusers/resources/ServiceUpdateOperationValidator.java

```diff
@@ -77,7 +77,11 @@ public ServiceUpdateOperationValidator(RequestValidations requestValidations) {
         if (FIELD_CUSTOM_BRANDING.equals(path)) {
             errors.addAll(checkIfValidJson(operation.get(FIELD_VALUE), FIELD_CUSTOM_BRANDING));
         } else if (FIELD_NAME.equals(path) || path.startsWith(FIELD_SERVICE_NAME_PREFIX)) {
-            requestValidations.checkIfExistsOrEmpty(operation, FIELD_VALUE).ifPresent(errors::addAll);
+            if (FIELD_NAME.equals(path) || path.endsWith('/' + SupportedLanguage.ENGLISH.toString())) {
+                requestValidations.checkIfExistsOrEmpty(operation, FIELD_VALUE).ifPresent(errors::addAll);
```

Comment:
I find the method name `checkIfExistsOrEmpty` sorely confusing. It seems like there is no validation error if the field value exists, but there is a validation error the field value is empty. But the method name is suggesting the presence or absence of a validation error for both scenarios. `failValidationIfEmpty` or `validateFieldValueIsNotEmpty` might be better.

Reply:
You’re right, it’s confusing; it was like that already. We’re going to (in a separate commit) rename this method to `checkExistsAndNotEmpty`, which should be clearer but still be consistent with the other methods.